doc/tutorials/using_windows_as_uos.rst

```diff
@@ -8,9 +8,9 @@ ACRN hypervisor.
 
 Verified version
 ****************
-* Clear Linux version: 32680
-* ACRN-hypervisor tag: v1.6 (acrn-2020w12.5-140000p)
-* ACRN-kernel tag: acrn-2020w12.5-140000p
+* Clear Linux version: 33050
+* ACRN-hypervisor tag: v1.6.1 (acrn-2020w18.4-140000p)
+* ACRN-Kernel (Service VM kernel): 4.19.120-108.iot-lts2018-sos
```

Comment:
That does not look like the right tag. I was expecting something like `acrn-2020wXX.X-14000p`. Is this really a tag that is (or will shortly be) valid in the `projectacrn/acrn-kernel` repo?

We added a tag called `v1.6` last time, I think it would be good to add a `v1.6.1` this time around too.